index.js

```diff
@@ -195,15 +195,21 @@ function fastifyAuth0Verify(instance, options, done) {
 
     // Setup our decorators
     instance.decorate('authenticate', authenticate)
-    instance.decorate('auth0Verify', auth0Options)
-    instance.decorateRequest('auth0Verify', auth0Options)
+    instance.decorate('auth0Verify', {
+      getter: () => auth0Options
+    })
```

Comment:
I don't think this is needed. Can you check?

Reply:
Ah good catch, removed.